three.js/src/threex/threex-armarkercontrols.js

```diff
@@ -266,10 +267,10 @@ ARjs.MarkerControls.prototype._initArtoolkit = function () {
         var sw, sh;
         var pscale, sscale;
         var w, h;
-        var pw, ph;
         var ox, oy;
-        vw = input_width; // need to setup input video width and height
-        vh = input_height;
+
+        vw = input_width = 640; // need to setup input video width and height
```

Comment:
This need to be fixed. I'm trying to add an EventListener to retrieve the video width and height as we did here https://github.com/kalwalt/AR.js/blob/2dd0dc8d2c98122cd7998103921f6ff95aa36fe9/three.js/src/threex/threex-armarkercontrols.js#L276-L283 but i get issues with video data.

Reply:
what you mean?

I guess the video input can be the video element? So the solution can be to take its sizes

Reply:
you are right, much simpler...